src/AWS.Deploy.CLI/ConsoleUtilities.cs

```diff
@@ -46,6 +48,59 @@ public ConsoleUtilities(IToolInteractiveService interactiveService, IDirectoryMa
             _directoryManager = directoryManager;
         }
 
+        public SortedSet<string> AskUserForList<T>(

```

Comment:
We should be able to mock the AWS calls and be able to have unit tests for this method.

Reply:
Done